src/audio/kpb.c

```diff
@@ -557,6 +557,14 @@ static int kpb_copy(struct comp_dev *dev)
 		}
 
 		copy_bytes = MIN(sink->free, source->avail);
+		if (!copy_bytes) {
```

Comment:
The last sentence of the commit description seems weird.

Reply:
Yes, corrected now.